beets/dbcore/queryparse.py

```diff
@@ -24,6 +24,10 @@
 PARSE_QUERY_PART_REGEX = re.compile(
     # Non-capturing optional segment for the keyword.
     r'(?:'
+    ur'(-|\u00ac)'  # Negation prefixes
```

Comment:
Since we have `unicode_literals` enabled, these are all Unicode already. Maybe we should change each piece to `ur` (or remove the `u` from this one) for uniformity.